include/openssl/crypto.h.in

```diff
@@ -389,9 +389,9 @@ int OPENSSL_isservice(void);
 
 void OPENSSL_init(void);
 # ifdef OPENSSL_SYS_UNIX
-void OPENSSL_fork_prepare(void);
-void OPENSSL_fork_parent(void);
-void OPENSSL_fork_child(void);
+DEPRECATEDIN_3_0(void OPENSSL_fork_prepare(void))
+DEPRECATEDIN_3_0(void OPENSSL_fork_parent(void))
+DEPRECATEDIN_3_0(void OPENSSL_fork_child(void))
```

Comment:
This works, but just to keep us on our toes @levitte recently changed the preferred way of doing this :-)

It should now be like this:

````
#  ifndef OPENSSL_NO_DEPRECATED_3_0
OSSL_DEPRECATEDIN_3_0 void OPENSSL_fork_prepare(void);
OSSL_DEPRECATEDIN_3_0 void OPENSSL_fork_parent(void);
OSSL_DEPRECATEDIN_3_0 void OPENSSL_fork_child(void);
#  endif
````